README.md

```diff
@@ -31,7 +39,46 @@ env:
 ```
 Please refer to our [troubleshooting section](docs/DEVELOPER_GUIDE.md#troubleshooting) for recommendations and tips.
 
-### Use ###
+### KFServing in 5 Minutes
+Make sure you have [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage),
+[kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux),
+[kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md) v3.5.4+,
+[helm 3](https://helm.sh/docs/intro/install) installed before you start.(2 mins for setup)
+1) Create a quick `kind` kubernetes local cluster.(this takes 30s)
+```bash
+kind create cluster
+```
+2) Install Istio lean version, Knative Serving, KFServing all in one.(this takes 30s)
```

Comment:
+ cert manager now, right?

Reply:
the quick install is not using cert manager, it is using Tommy's self signed cert gen script